app/controllers/application_controller.rb

```diff
@@ -5,6 +5,28 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, 'fwitter_time'
   end
 
-end
\ No newline at end of file
+  get '/' do
+    if logged_in?
+      redirect to "/tweets"
+    else
+      erb :"/index"
+    end
+  end
+
+  helpers do
+
+    def current_user
```

Comment:
a great thing here is to build an instance variable for `@current_user` that way we don't have to make an activeRecord query each time we load a page if it is set. 

example: 

```ruby
@current_user ||= User.find(session[:user_id]
```